README.md

```diff
@@ -1049,6 +1088,26 @@ On OS X and Linux, your git configuration file is stored in ```~/.gitconfig```.
     zap = fetch -p
 ```
 
+<a name="adding-empty-repository"></a>
+### I want to add an empty directory to my repository
+
+You can’t! git doesn’t support this, but there’s a hack. You can create a .gitignore file in the directory with the following contents:
+
+
+ # Ignore everything in this directory
```

Comment:
These should be in a codeblock (just not a `sh` codeblock). Indent them or add ticks around this.